plugin/core/panels.py

```diff
@@ -191,3 +191,20 @@ def run(self, edit: sublime.Edit, window_id: int) -> None:
                 if not erase_region.empty():
                     self.view.erase(edit, erase_region)
         clear_undo_stack(self.view)
+
+
+class LspClearLogPanelCommand(sublime_plugin.TextCommand):
+    def run(self, edit: sublime.Edit) -> None:
+        window = self.view.window()
+        if not window:
+            return
+        panel = ensure_server_panel(window)
+        if panel:
+            panel.run_command("lsp_clear_panel")
+
+    def is_visible(self) -> bool:
+        window = self.view.window()
+        if not window:
+            return False
+        panel = ensure_server_panel(window)
```

Comment:
Maybe not quite right for `is_visible` to create the panel?

To be fair I don't like that the panel is created with some delay. I would be fine with creating it as soon as possible, which this change basically achieves, but would want it more explicit. So you have two options:
 - change this code to not create the panel from `is_visible`
 - ensure that creation happens as soon as possible when LSP package is initializing

(Hopefully others share my opinion that it would be fine to create the panel early. Don't want to create back-and-forth situation here. :))

Reply:
Ok, created all panels on start.
c70552a

But If I open a new ST window, I will still be in a situation like before this commit c70552a